src/app/services/actions/query-action-creators.ts

```diff
@@ -16,21 +15,24 @@ export function queryResponseError(response: object): IAction {
 }
 
 export function runQuery(url: string): Function {
-  const headers = { Authorization: 'Bearer {token:https://graph.microsoft.com/}' };
-
   return (dispatch: Function) => {
     const respHeaders: any = {};
-
-    return fetch(`https://proxy.apisandbox.msdn.microsoft.com/svc?url=${url}`, { headers })
+    let authToken = '{token:https://graph.microsoft.com/}';
+    let graphUrl = `https://proxy.apisandbox.msdn.microsoft.com/svc?url=${url}`;
+    const authenticated = localStorage.getItem('authenticated');
+    if (authenticated) {
+      authToken = JSON.parse(authenticated).token;
+      graphUrl = url;
+    }
+    const headers = { Authorization: `Bearer ${authToken}` };
```

Comment:
We are supposed to be using the Graph Javascript SDK for making calls.  It has middleware to do this auth work.  We don't need to deal with tokens manually.

Reply:
See this sample here https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/dev/samples/browser/src/main.js#L13  (There is an issue with the latest release where this sample only works on classic Edge. This should be fixed soon.)